### PR TITLE
feat(wallet): Built out NFT Info Tooltip

### PR DESCRIPTION
**Pull request by Douglashdaniel**
## Description 
Built out the `NFT Info` tooltip for the new `Send` tab.

Resolves <https://github.com/brave/brave-browser/issues/26753>

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

1. Add an NFT to you portfolio
2. Go to brave://wallet/send
3. Select the `Send NFT` tab and click on `Select NFT`
4. Hover over the `info` icon to see the tooltip
5. NFT info should be accurate and the links too `Block explorers` should work.

https://user-images.githubusercontent.com/40611140/202002188-d77ac648-8971-4d1c-8e32-acc354df1441.mov

### CODE REVIEW

File: components/brave_wallet_ui/utils/string-utils.ts [hunk 1]
```diff
@@ -121,3 +121,13 @@ export const endsWithAny = (extensions: string[], url: string) => {
     return url.endsWith(suffix)
   })
 }
+
+export const getNFTTokenStandard = (token: BraveWallet.BlockchainToken) => {
```
Comment (josheleonard) [inline components/brave_wallet_ui/utils/string-utils.ts:125]: can we create a union or enum type for the token standards?

File: components/brave_wallet/browser/brave_wallet_constants.h [hunk 2]
```diff
@@ -689,6 +689,8 @@ constexpr webui::LocalizedString kLocalizedStrings[] = {
     {"braveWalletNFTDetailTokenStandard",
      IDS_BRAVE_WALLET_NFT_DETAIL_TOKEN_STANDARD},
     {"braveWalletNFTDetailTokenID", IDS_BRAVE_WALLET_NFT_DETAIL_TOKEN_ID},
+    {"braveWalletNFTDetaalContractAddress",
```
Comment (josheleonard) [inline components/brave_wallet/browser/brave_wallet_constants.h:692]:
```suggestion
    {"braveWalletNFTDetailContractAddress",
```

**Review by josheleonard — CHANGES_REQUESTED** [review]

File: components/brave_wallet_ui/stories/locale.ts [hunk 3]
```diff
@@ -644,6 +644,7 @@ provideStrings({
   braveWalletNFTDetailBlockchain: 'Blockchain',
   braveWalletNFTDetailTokenStandard: 'Token standard',
   braveWalletNFTDetailTokenID: 'Token ID',
+  braveWalletNFTDetaalContractAddress: 'Contract address',
```
[the remark this answers is not in the document]
Comment (Douglashdaniel) [reply components/brave_wallet_ui/stories/locale.ts]: Fixed!

File: components/brave_wallet_ui/utils/string-utils.ts (hunk 1 above)
Comment (Douglashdaniel) [reply components/brave_wallet_ui/utils/string-utils.ts:125]: Created a enum!

File: components/brave_wallet/browser/brave_wallet_constants.h (hunk 2 above)
Comment (Douglashdaniel) [reply components/brave_wallet/browser/brave_wallet_constants.h]: Fixed!

**Review by yrliou — APPROVED** [review]

**Pull request merged**